magefile.go

```diff
@@ -334,3 +337,13 @@ func PythonEnv() error {
 	_, err := mage.PythonVirtualenv(true)
 	return err
 }
+
+func BuildOpaBundle() error {
```

Comment:
Perhaps can be a good idea to add this function as a dependency to the build step
https://github.com/elastic/cloudbeat/blob/main/magefile.go?plain=1#L81. since without the rego bundle cb isn't really useable.